### DISCUSSION TITLE
PR to specific Branch

### DISCUSSION THREAD

**Discussion by zackrspv** [renovatebot/renovate#13395] [Request Help]
Is it possible to make it do the PR's automatically to a DIFFERENT branch while still monitoring the main branch? We ask this because we don't always want things, even minor updates, to hit master unless they hit our dev servers first.

**Comment by rarkins** [comment]
That is not technically possible because another branch could have completely different files to the default branch. So you can't monitor and target different branches (this is a git concept, not a Renovate limitation).

But you can *monitor* a different branch and target it too by configuring `baseBranches`

**Reply by zackrspv** [reply]
So our best bet would be to configure the baseBranch to use our STAGE, and allow auto merging there, and do our testing there before hitting master.
